.github/workflows/release.yml

```diff
@@ -1,90 +1,70 @@
+name: Publish plerkle plugin 
+
 on:
+  workflow_dispatch:
   push:
     tags:
-      - "v*"
+      - 'v*'
 
 env:
   CARGO_TERM_COLOR: always
-  IMAGE_NAME: plerkle-test-validator
-  RUST_VERSION: 1.69.0
-  SOLANA_VERSION_STABLE: v1.16.6
+
 jobs:
-  release-stable:
-    runs-on: buildjet-8vcpu-ubuntu-2004
+  build20:
+
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, ubuntu-22.04]
```

Comment:
Do we need the two OS matrix for releasing?  We don't need it to run for both right?

Reply:
Those should be the 2 os more used, and 22 builds won't work on 20 and viceversa. If you think that releasing for 22 only is best, it's an easy fix.

Reply:
Ok yes that makes sense for us to publish two.  @febo and I reviewed it more closely and had some questions on how it works.